src/pages/case-studies/[slug].astro

```diff
@@ -22,16 +16,15 @@ export async function getStaticPaths() {
 	}))
 }
 
-type Props = {
+export type Props = {
 	post: CollectionEntry<"caseStudies">
 }
 
-const { post } = Astro.props
+const { post } = Astro.props as { post: CollectionEntry<"caseStudies"> }
```

Comment:
Is this cast needed?

Reply:
No big either way if it's still broken 👍

Reply:
Ah ha, found it!  Looks like the language server is only automatically resolving `Astro.props` types if the `Prop` definition is at the top, not sure if that's expected or just a quirk of how it resolves the types + hoisted `getStaticPaths` definition